### PR TITLE
Add basic test for the ganglia input plugin

### LINKED ISSUE #6: No tests / Tests missing
Hi,
  this plugin has no test at all, this is a situation that should be amended soon, so for example changes in both the plugin or the logstash core himself, are properly tested in the CI environment.

If you are giving this plugin some test, feel free to update https://github.com/elastic/logstash/issues/3740 so we keep track of it.

### PR DESCRIPTION
**Pull request by purbon**
Fixes #6 

and the plugin that was broken with missing dependencies and wrong spec configuration.

### CODE REVIEW

File: logstash-input-ganglia.gemspec [hunk 1]
```diff
@@ -23,6 +23,11 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "logstash-core", '>= 1.4.0', '< 2.0.0'
 
   s.add_runtime_dependency 'logstash-codec-plain'
+  s.add_runtime_dependency 'logstash-filter-grok'
```
Comment (suyograo) [inline logstash-input-ganglia.gemspec]: @purbon why this dependency?

File: logstash-input-ganglia.gemspec [hunk 2]
```diff
@@ -23,6 +23,11 @@ Gem::Specification.new do |s|
...
+  s.add_runtime_dependency 'logstash-filter-date'
```
Comment (suyograo) [inline logstash-input-ganglia.gemspec]: same question here...

**Comment by purbon** [comment]
@suyograo clean up stale dependencies, I would say there were there as left overs, thanks for noticing they are not required anymore!

**Comment by purbon** [comment]
@suyograo is this in good shape to be merged? thanks a lot for your time.

**Pull request merged**